engine/src/vs_math.h

```diff
@@ -50,6 +50,5 @@
 #define FINITE(x) ( std::isfinite( x ) )
 #define ISNAN(x) ( std:: isnan( x ) )
 
-#define VS_PI 3.1415926535897931
```

Comment:
So I like moving from `VS_PI` to `M_PI` but from what I'm reading `M_PI` isn't standard (ANSI/ISO C/C++); do we have something checking for it to exist at the CMake level and if not defining it? How we define `version.h` could probably be a good method.

Reply:
I checked and it is. You just need some kind of magic pre-processor for it to be included. I'll add that to the list of things to do and reasons why windows annoys me :)

Reply:
So I spent a few minutes trying to figure this out and then stopped. The game builds on windows.

Reply:
when I was reviewing this, I know I was seeing an error around `M_PI` for at least one of the various PRs out there....but yes, this PR does build on all the platforms. May be there was a hiccup in the GH workers?